Tests/Indicators/CommonIndicatorTests.cs

```diff
@@ -234,7 +234,16 @@ protected virtual void RunTestIndicator(IndicatorBase<T> indicator)
         /// </summary>
         protected virtual Action<IndicatorBase<T>, double> Assertion
         {
-            get { return (indicator, expected) => Assert.AreEqual(expected, (double) indicator.Current.Value, 1e-3); }
+            get
+            {
+                return (indicator, expected) =>
+                {
+                    Assert.AreEqual(expected, (double)indicator.Current.Value, 1e-3);
+
+                    var relativeDifference = Math.Abs(((double)indicator.Current.Value - expected) / expected);
+                    Assert.LessOrEqual(relativeDifference, 1); // less than 1% error rate
```

Comment:
That's a quite "big" relative difference. The bigger the value is the lesser luck to catch a bug you have. What about 0.1 or 0.01?

Reply:
Yes agree 💯, that's just the current minimum to avoid failures with existing tests & indicators